config/locales/doc_auth/zh.yml

```diff
@@ -148,6 +149,7 @@ zh:
         pretending to be you. %{link_html}'
       getting_started_learn_more: Learn more about verifying your identity
       how_to_verify: 你有在网上验证身份或者到参与邮件亲身验证身份的选择
+      how_to_verify_selfie: 你可以选择用你的手机在网上验证身份或者到参与本项目的邮局亲身验证身份
```

Comment:
the previous comment should have been for `how_to_verify_selfie`. This is what I'm seeing: ```<Conditional [[[Body]]] for biometric flow: “你可以选择用你的手机在网上验证身份或者到参与本项目的邮局亲身验证身份。
”>
```

Reply:
Ah, yes! Thank you!!  Resolved with commit abc13c8910093067c70814904d0aeddc1415ec78